openwrt-openwisp-monitoring/files/monitoring.init

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+STOP=15
+USE_PROCD=1
+PROG="/usr/sbin/openwisp_monitoring"
+PROG_NAME="OpenWISP monitoring daemon"
+
+
+config_openwisp() {
+    config_get base_url "$1" url
+    config_get uuid "$1" uuid
+    config_get key "$1" key
+    config_get_bool verify_ssl "$1" verify_ssl "0"
+    config_get respawn_threshold "$1" respawn_threshold "3600"
+    config_get respawn_timeout "$1" respawn_timeout "5"
+    config_get respawn_retry "$1" respawn_retry "5"
+
+    if [ $base_url ]; then base_url="--url $base_url"; fi
+    if [ $uuid ]; then uuid="--uuid $uuid"; fi
+    if [ $key]; then key="--key $key"; fi
+    if [ $verify_ssl ]; then verify_ssl="--verify_ssl $verify_ssl"; fi
+}
+
+config_monitoring() {
+    config_get monitored_interfaces "$1" monitored_interfaces "*"
+    config_get interval "$1" interval "300"
+    interval="--verify_ssl $verify_ssl"
+    monitored_interfaces="--monitored_interfaces $monitored_interfaces"
+
+    procd_open_instance "openwisp_monitoring_${1}"
+    procd_set_param command $PROG $uuid $key $base_url $verify_ssl $monitored_interfaces
+    procd_set_param respawn ${respawn_threshold} ${respawn_timeout} ${respawn_retry}
+    procd_close_instance
+    logger -s "$PROG_NAME started"
+}
+
+start_service() {
+        config_load openwisp
+        config_foreach config_openwisp http
+        config_load monitoring
+        config_foreach config_monitoring monitoring
+}
+
+stop_service() {
+    logger -s "$PROG_NAME stopping"
+    echo stop
```

Comment:
Not needed

Reply:
Modified and added a tag too as done in [openwisp-config](https://github.com/openwisp/openwisp-config/blob/7cbb2eb2f07a6bdbff87c8034d7bf22efc510c6e/openwisp-config/files/openwisp.init#L105)